### PR TITLE
fix: adjust discussed feed styles to match new mobile UX

### PR DESCRIPTION
**Pull request by DragosIorgulescu**
### New UI

Screenshot 2024-04-01 at 14 15 40

### Previous UI

Screenshot 2024-04-02 at 12 24 32

---

### New UI

Screenshot 2024-04-01 at 14 14 12

### Previous UI

Screenshot 2024-04-02 at 12 24 53

## Events

N / A

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-252

### CODE REVIEW

File: packages/shared/src/components/CommentFeed.tsx [hunk 1]
```diff
@@ -103,10 +103,7 @@ export default function CommentFeed<T>({
         isFetchingNextPage={queryResult.isFetchingNextPage}
         canFetchMore={checkFetchMore(queryResult)}
         fetchNextPage={queryResult.fetchNextPage}
-        className={classNames(
-          'w-full',
-          isNewMobileLayout && isMainFeed && 'px-4',
```
Comment (sshanzel) [inline packages/shared/src/components/CommentFeed.tsx:108]: What should this look like in control values? Can you also attach screenshot of the old UI? Just to make sure everything still looks normal.

**Review by sshanzel — APPROVED** [review]
Looks great. Kindly verify that the control still looks normal, we should be good here :shipit:

**Review by sshanzel — CHANGES_REQUESTED** [review]
Sorry for the late request, I think the labels on the navigation for tablet is a bit bigger than expected.

**Comment by DragosIorgulescu** [comment]
> Sorry for the late request, I think the labels on the navigation for tablet is a bit bigger than expected.

@sshanzel not sure what you are referring to, could you elaborate? Which labels?

**Comment by sshanzel** [comment]
@DragosIorgulescu hi, sorry for the lack of clarity. These ones. In the Figma they're at `typo-caption2`, after inspecting the element, I think they're on `typo-callout`.

Screenshot 2024-04-02 at 5 28 59 PM

**Comment by DragosIorgulescu** [comment]
> @DragosIorgulescu hi, sorry for the lack of clarity. These ones. In the Figma they're at typo-caption2, after inspecting the element, I think they're on typo-callout.

@sshanzel thank you for the clarification 🙏  I updated this, however to me these look way tinier than designs now (even though I agree these are specified with `typo-caption2` there. See below

Screenshot 2024-04-02 at 13 14 25

This feels like we should use `typo-caption1` rather 🤔

**Comment by sshanzel** [comment]
> This feels like we should use typo-caption1 rather

Yep, totally agree with this. Sometimes it really looks different from Figma, I think what you suggested it the sweet spot.

**Review by sshanzel — APPROVED** [review]
Feel free to change it to `typo-caption1` - got nothing more on this, looks amazing! :shipit:

**Pull request merged**